packages/react-storage/src/components/StorageBrowser/subcomponents/PaginationControl/PaginationControl.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { classNames } from '@aws-amplify/ui';
+import { NavElementProps } from '@aws-amplify/ui-react/internal';
+import { useElement } from '../../context/elements';
+import { NextButton } from './NextButton';
+import { PreviousButton } from './PreviousButton';
+import { CurrentPage } from './CurrentPage';
+import { Item } from './Item';
+
+const PaginationControlPrimitive = <T extends NavElementProps>({
+  ariaLabel = 'Pagination',
+  className,
+  children,
+  ...rest
+}: T): JSX.Element => {
+  const Nav = useElement('Nav');
+
+  return (
+    <Nav
+      {...rest}
+      aria-label={ariaLabel}
+      className={classNames(
```

Comment:
I'm assuming we want to do _something_ like this for elements in this base version that need to provide some layout styling but I haven't started adding those basic layout styles.